apache-maven/src/assembly/shared/run.ps1

```diff
@@ -0,0 +1,13 @@
+& $JAVACMD `
+  $MAVEN_OPTS `
+  $MAVEN_DEBUG_OPTS `
+  -classpath $LAUNCHER_JAR `
```

Comment:
Will this handle spaces in `$LAUNCHER_JAR`?

Reply:
To my knowledge, yes. You don't expect spaces to be handled?

Reply:
I would expect, at least the same behavior as the others...